src/Program.cs

```diff
@@ -311,7 +311,18 @@ private static async Task<string> GetCheckSaveRunCommandContentAsync(RunCommand
         try
         {
             ConsoleHelpers.PrintStatus($"Executing: {command.ScriptToRun} ...");
-            var finalContent = await GetFinalRunCommandContentAsync(command);
+            
```

Comment:
No, you don't need to change this method at all... just keep calling the `GetFinalRunCommandContentAsync` method... that will call `GetFormattedRunCommandContentAsync`... Its in there that you can pass the timeout (from the command.Timeout property) to the RunShellCommandAsync method.